src/config/config_melbourne.yml

```diff
@@ -0,0 +1,87 @@
+# City name
+city: Melbourne, VIC, Australia
+# City centerpoint latitude & longitude (default geocoded values set)
+city_latitude: -37.8163
+city_longitude: 144.9631
+
+# City's time zone: defaults to the local time zone of computer initializing the city's config file
+timezone: Australia/Melbourne
+
+# Radius of city's road network from centerpoint in km, required if OSM has no polygon data (defaults to 20km)
+city_radius: 50
+
+# The folder under data where this city's data is stored
+name: melbourne
+
+# If given, limit crashes to after startdate and no later than enddate
+# Recommended to limit to just a few years for now
+startdate: 1/8/2013
+enddate:
+
+
+#################################################################
+# Configuration for data standardization
+
+# crash file configurations
+crashes_files:
+  C:\Users\superuser\Documents\crash_prediction\data4democracy\crash-model\data\melbourne\raw\crashes\Crashes_Last_Five_Years.csv:
```

Comment:
As per the configs for other cities, you should be able to specify just the filename for your data sources, as the pipeline expects to find them in a predefined location.

Brisbane crash data example:
```
crashes_files:
  locations_2014_2017.csv:
```